rosdep/base.yaml

```diff
@@ -1656,13 +1656,22 @@ libgps:
   ubuntu: [libgps-dev]
 libgsl:
   arch: [gsl]
-  debian: [libgsl0-dev]
+  debian:
+    wheezy: [libgsl0-dev]
+    jessie: [libgsl0-dev]
+    stretch: [libgsl-dev]
+    sid: [libgsl-dev]
```

Comment:
We generally prefer not to put sid rules in since it's a rolling distro and thus is expected to change and we don't want to be trying to keep up.